### PR TITLE
Output status updates to stderr

### PR DESCRIPTION
**Pull request by dev-zero**
This makes it possible to easily redirect the actual output of e.g. "bios get" to a file

### CODE REVIEW

**Comment by dev-zero** [comment]
But maybe it would be to introduce a logging library, what do you think @rustydb ?

File: internal/query/query.go [hunk 1]
```diff
@@ -44,7 +45,7 @@ func Async(fn func(host string) interface{}, hosts []string) map[string]any {
 
 	v := viper.GetViper()
 	if !v.GetBool("json") {
```
Comment (rustydb) [inline internal/query/query.go:47]: I think we could do one better, and remove this conditional throughout both `query.go` and `set.go`. The conditional's intention was to ensure `--json` didn't output any non-JSON, but if this output is always directed to `os.Stderr` then it doesn't matter.

**Review by rustydb — APPROVED** [review]
I left one comment but I am approving this in the meantime, my comment is not a merge-blocking request.

**Comment by rustydb** [comment]
> But maybe it would be to introduce a logging library, what do you think @rustydb ?

Not opposed, presuming it could log to a terminal session as well as a file respective of the OS.

**Review by jacobsalmela — APPROVED** [review]

File: internal/query/query.go (hunk 1 above)
Comment (rustydb) [reply internal/query/query.go:47]: I'll tackle this in #50

**Pull request merged**